docs/development/adding-a-package-manager.md

```diff
@@ -67,3 +77,17 @@ This is used when more than one package manager shares settings from a common la
 ### `supportsLockFileMaintenance` (optional)
 
 Set to true if this package manager needs to update lock files in addition to package files.
+
+### `updateArtifacts` (async, optional)
+
+This function is triggered if a dependency has been updated for a package file or `lockfileMaintenance` has been triggered.
```

Comment:
```suggestion
This function triggers:

- after a dependency update (for a package file), or
- after `lockfileMaintenance`
```

This reads better to me, but it may be technically wrong. Please check before applying my suggestion. 😉